ibm/data_source_ibm_certificate_manager_certificate_details.go

```diff
@@ -0,0 +1,168 @@
+package ibm
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataIBMCertificateManagerCertificateDetails() *schema.Resource {
+	return &schema.Resource{
+		Read: dataIBMCertificateManagerCertificateDetailsRead,
+		Schema: map[string]*schema.Schema{
+			"cert_id": {
+				Type:     schema.TypeString,
```

Comment:
Instead of ID can we have parameters to read the details of certificate absed on certificate name and certifcate instance